engine/schema/src/main/java/com/cloud/network/as/dao/AutoScaleVmGroupVmMapDaoImpl.java

```diff
@@ -67,7 +67,7 @@ public int countAvailableVmsByGroup(long vmGroupId) {
         SearchCriteria<Integer> sc = CountBy.create();
         sc.setParameters("vmGroupId", vmGroupId);
         sc.setJoinParameters("vmSearch", "states",
-                State.Starting, State.Running, State.Stopping, State.Migrating);
+                State.Starting, State.Running, State.Stopping, State.Migrating, State.Error, State.Stopped);
```

Comment:
This may make autoscaler to not retry deployment if any deployment goes into error state intermittently. Should we include Error state after some n reties?

Reply:
Yeah, that was my worry too, wanted some inputs on it. I'll do that. Thanks @shwstppr